docs/docs/tutorials/codealong/contract_tutorials/nft_contract.md

```diff
@@ -0,0 +1,378 @@
+
+## How this will work
+
+Before writing the functions, let's go through them to see how this contract will work:
+
+### Initializer
+
+There is one `initializer` function in this contract, and it will be selected and executed once when the contract is deployed, similar to a constructor in Solidity. This is marked `public`, so the function logic will be transparent.
+
+### Public functions
+
+These are functions that have transparent logic, will execute in a publicly verifiable context and can update public storage.
+
+- [`constructor`](#constructor) - executed when the contract instance is deployed
+- [`set_admin`](#set_admin) - updates the `admin` of the contract
+- [`set_minter`](#set_minter) - adds a minter to the `minters` mapping
+- [`mint`](#mint) - mints an NFT with a specified `token_id` to the recipient
+- [`transfer_in_public`](#transfer_in_public) - publicly transfer the specified token
+- [`finalize_transfer_to_private`](#finalize_transfer_to_private) - finalized a transfer to from the public context to the private context (more on this below)
+
+#### Public `view` functions
+
+These functions are useful for getting contract information for use in other contracts, in the public context.
+
+- [`public_get_name`](#public_get_name) - returns name of the NFT contract
+- [`public_get_symbol`](#public_get_symbol) - returns the symbols of the NFT contract
+- [`get_admin`](#get_admin) - returns the `admin` account address
+- [`is_minter`](#is_minter) - returns a boolean, indicating whether the provided address is a minter
+- [`owner_of`](#owner_of) - returns the owner of the provided `token_id`
+
+### Private functions
+
+These are functions that have private logic and will be executed on user devices to maintain privacy. The only data that is submitted to the network is a proof of correct execution, new data commitments and nullifiers, so users will not reveal which contract they are interacting with or which function they are executing. The only information that will be revealed publicly is that someone executed a private transaction on Aztec.
+
+- [`transfer_to_private`](#transfer_to_private) - initiates the transfer of an NFT from the public context to the private context
+- [`prepare_private_balance_increase`](#prepare_private_balance_increase) - creates a [partial note](../../../aztec/concepts/storage/partial_notes.md) to transfer an NFT from the public context to the private context.
+- [`cancel_authwit`](#cancel_authwit) - emits a nullifier to cancel a private authwit
+- [`transfer_in_private`](#transfer_in_private) - transfers an NFT to another account, privately
+- [`transfer_to_public`](#transfer_to_public) - transfers and NFT from a private accounts' balance to a specified accounts' public balance
```

Comment:
```suggestion
- [`transfer_to_public`](#transfer_to_public) - transfers a NFT from private to public context
```

Reply:
in other method descriptions we don't specify that you can transfer to a different address so i am removing it for cohesion.